### ISSUE TITLE
Cordova 8 build issues

### ISSUE THREAD

**Issue by jeremyspatrick** [PSPDFKit/Cordova-Android#25]
1. plugin.xml needs the following change

line 51

instead of target="res/xml/config.xml"
it should be target="app/src/main/res/xml/config.xml"

```
<config-file parent="/*" target="app/src/main/res/xml/config.xml">
            <feature name="PSPDFKitCordovaPlugin">
                <param name="android-package" value="com.pspdfkit.cordova.PSPDFKitCordovaPlugin"/>
            </feature>
        </config-file>
```

2. the .aar file should go in /platforms/android/app/libs instead of /platform/android/libs

**Comment by radazzouz** [comment]
@jeremyspatrick Thanks for reporting!

**Comment by davidschreiber** [comment]
This issue will be resolved by #26. Thanks again for bringing this to our attention.
